packages/sdk/src/index.ts

```diff
@@ -1,11 +1,17 @@
-export { getVincentToolClient } from './tool';
-export { jwt } from './jwt';
-export { getVincentWebAppClient } from './app';
-
+import * as jwt from './jwt';
+export { jwt };
 export type { JWTConfig, VincentJWT, VincentJWTPayload } from './jwt/types';
+
+export { getVincentToolClient } from './tool';
 export type { VincentToolParams, VincentToolClientConfig, VincentToolClient } from './tool/types';
+
+export { getVincentWebAppClient } from './app';
 export type {
   VincentWebAppClient,
   VincentAppClientConfig,
   RedirectToVincentConsentPageParams,
 } from './types';
+
+import * as expressAuthHelpers from './express-authentication-middleware';
+export { expressAuthHelpers };
+export type { ExpressAuthHelpers } from './express-authentication-middleware/types';
```

Comment:
Exporting the types needed for consumers to define AuthenticatedRequestHandler functions as an interface to avoid cluttering the main API export with Express-specific types.